api/document/management/commands/import_xml_doc.py

```diff
@@ -18,6 +18,8 @@ def fetch_policy(identifier: str):
 
 
 def import_xml_doc(policy, xmlstream):
+    policy.workflow_phase = WorkflowPhases.published.name
+    policy.save()
```

Comment:
Do you think should should happen after the `serializer.save()` step? Alternatively, we could wrap the whole thing in a transaction.

Reply:
Good idea, I just wrapped it all in a transaction!